LunaDll/Input/LunaGameController.cpp

```diff
@@ -486,6 +486,26 @@ std::string LunaGameControllerManager::getSelectedControllerName(int playerNum)
     return "Keyboard";
 }
 
+int LunaGameControllerManager::getSelectedControllerStickX(int playerNum)
+{
+    LunaGameController* controller = getController(playerNum);
+    if (controller != nullptr)
+    {
+        return controller->getStickX();
+    }
+    return 0;
+}
+
+int LunaGameControllerManager::getSelectedControllerStickY(int playerNum)
+{
+    LunaGameController* controller = getController(playerNum);
+    if (controller != nullptr)
+    {
+        return controller->getStickY();
+    }
+    return 0;
+}
+
```

Comment:
Defining two separate functions for the X axis and the Y axis seems kinda redundant. Since you're calling these functions at the same time in `LunaLuaGetSelectedControllerStickPosition`, that means that you're going to call `getController` and check whether its result is `nullptr` or not twice.